### ISSUE TITLE
App registration requires varification by Google

### ISSUE THREAD

**Issue by nathhad** [glotlabs/gdrive#35]
After following the directions to create new API credentials, I received the following message that the app would require verification (see screenshot).

![Screenshot_2023-04-07_15-42-28](https://user-images.githubusercontent.com/12617618/230668287-aab3491e-5e71-418f-aa39-3697793e8f49.png)

The process is very non-trivial. Is this a recent change? Is there a workaround in the instructions? Or have I simply followed the process incorrectly? Thanks!

**Comment by nathhad** [comment]
Update - so far gdrive seems to be working fine despite this message. It does generate a scare screen when you first connect. It's possible this scare screen is the only issue. It may just be that the warning and scare screen just need to be described at the end of https://github.com/glotlabs/gdrive/blob/main/docs/create_google_api_credentials.md with a note that it won't affect operation, and not to worry about it. I will update if I see it cause any actual issues.

**Comment by prasmussen** [comment]
Yeah, I have the same warning, but it has been working fine all along. Adding a note about it in the doc is a good suggestion.

**Comment by dommilosz** [comment]
You don't need to publish the app at all. Publishing allows other users than added ones to use the app. In testing mode it's still possible to use the app with added testing users.

**Comment by Glich440** [comment]
I think that if you don't publish it the key will become invalid after a week
